src/ds5/ds5-auto-calibration.cpp

```diff
@@ -10,6 +10,13 @@
 #include "librealsense2/rsutil.h"
 #include "../algo.h"
 
+#undef UCAL_DEBUG
+#ifdef UCAL_DEBUG
+#define LOG_OCC_DEBUG(...)   do { CLOG(WARNING   ,"librealsense") << __VA_ARGS__; } while(false)
```

Comment:
It is weird that the macro is called debug, and the logs are with warnings severity - please consider changing one of them

Reply:
I'll rename to LOG_OCC_WARN even though it is actually used for debugging only.
The reason I didn't used Debug level as it generates way too much non-related messages (backend, mostly) that make it impossible to review.
We need to think of a more granular debug level usage..

Reply:
ok